src/components/about/About.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Footer from '../footer/Footer.tsx'
+import Nav from './../nav/Nav.tsx'
```

Comment:
lets change the naming from Nav to something more descriptive?

Reply:
changed to MainPageNavigation